docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -61,21 +61,42 @@ p:not([class]) {
   margin-top: 0;
 }
 
+.field-hs-news-link a {
```

Comment:
Let's move the styles for `.field-hs-news-link a` into `docroot/themes/humsci/humsci_basic/src/scss/components/_node--news-item.scss` alongside the other `.field-hs-news` classes.